realm/realm-jni/build.gradle

```diff
@@ -75,7 +75,8 @@ def toolchains = [
 ]
 
 def allTargets = [
-    new Target( name:'arm',     abi:'armeabi',     toolchain:toolchains.find {it.name == 'arm'},    cflags:[ '-mthumb' ] ),
+    // gcc for armeabi has problem to compile std::excpetion_ptr. Disable this ABI for now. We might drop it or use the new NDK clang to compile it.
+    //new Target( name:'arm',     abi:'armeabi',     toolchain:toolchains.find {it.name == 'arm'},    cflags:[ '-mthumb' ] ),
```

Comment:
> clang seems to support `exception_ptr` but we are still investigating.

Never mind about my question. I almost forgot the point. Thanks for the reminder. 👍